package.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<package>
+  <name>v4r</name>
+  <version>1.0.0</version>
```

Comment:
not sure this is the correct version number. I took it from [v4r.pc.in](https://github.com/strands-project/v4r/blob/master/v4r.pc.in).